src/test/java/test/MyAccountPageDisplayedTest.java

```diff
@@ -0,0 +1,51 @@
+package test;
+
+import Utils.CommonClass;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.Assert;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+import pages.HomePage;
+import pages.LogInPage;
+import pages.MyAccountPage;
+
+@Test
+public class MyAccountPageDisplayedTest extends CommonClass {
+
+    HomePage homepage;
+    WebDriver driver;
+    LogInPage loginPage;
+    MyAccountPage myAccountPage;
+
+    @BeforeClass
+    public void setup() throws InterruptedException{
```

Comment:
This class and LoginTest class have a part which is the same for all of them: the constructor and the driver. How can you update the code so these classes don't repeat?